src/app/api/common/metrics/getMetrics.ts

```diff
@@ -5,12 +5,22 @@ import { cache } from "react";
 async function getMetrics() {
   const { namespace, contracts } = Tenant.current();
   const totalSupply = await contracts.token.contract.totalSupply();
-  const votableSupply = await prisma[`${namespace}VotableSupply`].findFirst({});
 
-  return {
-    votableSupply: votableSupply?.votable_supply || "0",
-    totalSupply: totalSupply.toString(),
-  };
+  try {
+    const votableSupply = await prisma[`${namespace}VotableSupply`].findFirst(
+      {}
+    );
+    return {
+      votableSupply: votableSupply?.votable_supply || "0",
+      totalSupply: totalSupply.toString(),
+    };
+  } catch (e) {
+    // Handle prisma errors for new tenants
```

Comment:
This handles the new tenant edge case. Basically return a zero before votable supply is indexed.